graphql/version_resolver.go

```diff
@@ -469,6 +470,27 @@ func (r *versionResolver) VersionTiming(ctx context.Context, obj *restModel.APIV
 	}, nil
 }
 
+// ExternalPatchLinks is the resolver for the externalPatchLinks field.
+func (r *versionResolver) ExternalPatchLinks(ctx context.Context, obj *restModel.APIVersion) ([]*ExternalPatchLink, error) {
+	pRef, err := data.FindProjectById(*obj.Project, false, false)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding project `%s`: %s", *obj.Project, err.Error()))
+	}
+	if pRef == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Project `%s` not found", *obj.Project))
+	}
+	var externalPatchLinks []*ExternalPatchLink
+	for _, link := range pRef.ExternalLinks {
+		// replace {version_id} with the actual version id
```

Comment:
Does it make sense to support any other fields?

Reply:
I was thinking author's name or branch or some other relevant information but we can add those in later if we need them.